src/bikeRental/index.js

```diff
@@ -1,8 +1,18 @@
 // @flow
+import geolib from 'geolib'
```

Comment:
Fikk du sett noe på http://turfjs.org/ for lettere utregninger av kartoperasjoner?

Reply:
Nei, gjorde meg ferdig med denne da alt nesten var klart. Kanskje det var noe som

Reply:
Okei, men ta og sjekk da om turfjs vil være lettere å forholde seg til en geolib. Tenker også at det er greit at vi bruker de samme bibliotekene rundt om i kodebasen vår ;)

Reply:
Høres ut som en ganske fair deal 🤝